### PR TITLE
remove flake8-commas from recommended due to bugs in the plugin

### PR DESCRIPTION
**Pull request by jensens**
see:

- https://github.com/flake8-commas/flake8-commas/issues/43
-  https://github.com/plone/plone.api/pull/377

### CODE REVIEW

**Comment by jensens** [comment]
If merged also remove at https://github.com/plone/buildout.coredev/blob/5.1/experimental/qa.cfg#L26

**Comment by gforcada** [comment]
I'm *personally* against it, as breaking statements into more than one line rather than cramming everything and the kitchen sink into one line is far from readable.

Two highly recommended readings/watchings on the topic:
- http://treyhunner.com/2017/07/craft-your-python-like-poetry/
- https://www.youtube.com/watch?v=knMg6G9_XCg

Now I notice that is the very same guy 😆

File: CHANGES.rst [hunk 1]
```diff
@@ -4,6 +4,9 @@ Change history
 2.3 (unreleased)
 ----------------
 
+- remove ``flake8-commas`` from recommended due to bugs in the plugin.
```
Comment (hvelarde) [inline CHANGES.rst]: "Remove"

**Comment by jensens** [comment]
@gforcada ok to remove it unless the reported issues are fixed in the plugin? It does not help to get false errors reported.

**Comment by gforcada** [comment]
@jensens the plone.api issue was merged without needing to remove this plugin. Actually, thanks to @loechel we have a much more readable code now on plone.api.

He had to add a single ``noqa: C816``, compared to the massive amounts of lines he cleaned up.

Although I don't like to litter the code base with noqa, it is far better to leave some here and there rather than removing the plugins (IMHO of course).

**Comment by graingert** [comment]
If you need to support versions of python lower than 3.6 you should ignore C816 globally @gforcada

**Pull request closed without merging**